projects/v2/src/app/components/info/info.component.html

```diff
@@ -33,6 +33,15 @@
           {{info.notes}}
         </div>
       </div>
+      <div *ngIf="info?.references.length">
```

Comment:
Need to account for when no references like is done in the CT nodes

Reply:
Ie like this block is handling references... https://github.com/hubmapconsortium/ccf-asct-reporter/pull/335/files#diff-e6750482d045d64132a8fa55b37b0518440c8400225448511bc3e103b7f80940R91